active/NumberBox/NumberBox.md

```diff
@@ -118,28 +93,22 @@ with a "///" comment above the member or type. -->
 | StepFrequency | Gets or sets the value part of a value range that steps should be created for. |
 
 ## API Details
-<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+<!-- todo: Missing some APIs in here, like TemplatSettings class and event args classes -->
```

Comment:
I added this for a moment, but was unsure of what could or should be templated here. @MikeHillberg or @teaP, did you have something particular in mind?

Reply:
Handled offline. Events updated and TemplateSettings unneeded. Resolving.